src/w2utils.js

```diff
@@ -1416,37 +1416,62 @@ class Utils {
                     text = this.execTemplate(text, actions)
                 }
                 let html = `
-                    <div id="w2ui-notify">
-                        <div class="${options.class} ${options.error ? 'w2ui-notify-error' : ''}">
+                    <div id="w2ui-notify" class="${options.where === document.body ? 'fixed-notify': 'absolute-notify'} ${options.side}">
+                        <div class="${options.error ? 'w2ui-notify-error' : ''}">
                             ${text}
                             <span class="w2ui-notify-close w2ui-icon-cross"></span>
                         </div>
                     </div>`
                 query(options.where).append(html)
                 query(options.where).find('#w2ui-notify').find('.w2ui-notify-close')
                     .on('click', event => {
-                        query(options.where).find('#w2ui-notify').remove()
-                        resolve()
+                        this.removeNotify()
                     })
                 if (options.actions) {
                     query(options.where).find('#w2ui-notify .w2ui-notify-link')
                         .on('click', event => {
                             let value = query(event.target).attr('value')
                             options.actions[value]()
-                            query(options.where).find('#w2ui-notify').remove()
-                            resolve()
+                            this.removeNotify()
                         })
                 }
                 if (options.timeout > 0) {
                     this.tmp.notify_timer = setTimeout(() => {
-                        query(options.where).find('#w2ui-notify').remove()
-                        resolve()
+                        this.removeNotify()
                     }, options.timeout)
                 }
             }
         })
     }
 
+    /**
+    * Removes the currently active notification from the DOM and cleans up associated resources.
+    * 
+    * - Resolves any pending Promise linked to the notification.
+    * - Clears the timeout associated with the notification, if one exists.
+    * - Removes the notification element (`#w2ui-notify`) from its parent container.
+    * - Resets all temporary references (`tmp`) to ensure proper cleanup.
+    * 
+    * This method ensures that any active notification is properly terminated, 
+    * preventing memory leaks or conflicts with future notifications.
+    */
+
```

Comment:
I do not like that this is part of `w2utils`, the only purpose of this is to clean up existing notify, which is also done if you call notify without parameters. Can we move it inside `notify` function and rename it to `_removeNofity` to indicate that it is internal. So, `w2utils.notify(msg)` displays a message and `w2utils.nofity()` hides it